test/chain.js

```diff
@@ -2,28 +2,47 @@ var bitcoin = require('bitcoinjs-lib')
 var Chain = require('../chain')
 var test = require('tape')
 var fixtures = require('./fixtures/chain')
+var createKeccakHash = require('keccak')
+
+function segwitAddr (node) {
+  var hash = bitcoin.crypto.hash160(node.getPublicKeyBuffer())
+  var script = bitcoin.script.witnessPubKeyHash.output.encode(hash)
+  return bitcoin.address.fromOutputScript(script)
+}
+
+function ethAddr (node) {
+  return '0x' + createKeccakHash('keccak256')
+    .update(node.getPublicKeyBuffer())
+    .digest().toString('hex')
+}
```

Comment:
A notable side-effect of how this could work

Reply:
@dcousens I do not understand, why not `digest('hex')`?

Reply:
> digest('hex')

Forgot that was an option - any advantage?